ServerCore/Pages/Shared/_EventNavigationPartial.cshtml

```diff
@@ -205,7 +205,7 @@
                             <li class="nav-item"><a style="color:yellow" asp-page="/Swag/Register" title="Lunch">Pick Lunch by @(Event.TeamMiscDataChangeEnd.ToString("M/dd"))!</a></li>
                         }
                     }
-                    else if (Event.EventHasTeamSwag)
+                    else if (Event.EventHasTeamSwag && isOnTeam)
```

Comment:
The block above should arguably have this check as well (if you sign up for lunch and aren't on a team, you're not getting lunch)

Reply:
For individual lunches, I think it makes sense this way.